lib/dat-worker-pool/runner.rb

```diff
@@ -0,0 +1,125 @@
+require 'system_timer'
+require 'thread'
+require 'dat-worker-pool'
+
+class DatWorkerPool
+
+  class Runner
```

Comment:
@kellyredding - All of the runner logic was just pulled from the dat worker pool, I didn't change it how it spawns or despawns workers or how it starts or shutsdown.